guides/common/modules/proc_distributing-archived-content-view-versions.adoc

```diff
@@ -1,5 +1,5 @@
 [id="Distributing_Archived_Content_View_Versions_{context}"]
-= Distributing Archived Content View Versions
+= Distributing archived content view versions
 
 The setting **Distribute archived content view versions** enables hosting of non-promoted Content View version repositories in the {Project} content web application along with other repositories.
```

Comment:
```suggestion
The setting *Distribute archived content view versions* enables hosting of non-promoted Content View version repositories in the {Project} content web application along with other repositories.
```

unrelated to your PR; double asterisks should not be necessary here.

Reply:
fixed. I also capitalized Content View, since we seem to be following that convention. The UI should be checked.